main.rb

```diff
@@ -0,0 +1,144 @@
+require './person'
+require './book'
+require './classroom'
+require './rental'
+require './student'
+require './teacher'
+# rubocop:disable all
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_books
+    @books.each do |book|
+      puts "Title: #{book.title} , Author: #{book.author}"
+    end
+  end
+
+  def all_people
+    @people.each do |person|
+      puts "[#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person
+    puts 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+    person_type = gets.chomp
+
+    if person_type != '1' && person_type != '2'
+      puts 'Invalid option'
+      nil
+    end
+
+    print 'Age:'
+    age = gets.chomp
+
+    print 'Name:'
+    name = gets.chomp
+
+    person =
+      case person_type
+      when '1'
+        print 'Has parent permission? [Y/N]: '
+        parent_permission = gets.chomp
+        parent_permission = parent_permission.downcase == 'y'
+
+        Student.new(age, name, parent_permission)
+      when '2'
+        print 'Specialization:'
+        specialization = gets.chomp
+
+        Teacher.new(age, specialization, name)
+      end
+
+    @people << person
+    puts 'Person created'
+  end
+
+  # *Create a book.
+  def create_book
+    print 'Title: '
+
+    title = gets.chomp
+
+    print 'Author: '
+
+    author = gets.chomp
+
+    @books << Book.new(title, author)
+    puts 'Book created'
+  end
+
+  def create_rental
+    print 'Select a book from the following list by number'
+    @book.each_with_index do |book, index|
```

Comment:
- [ ] The instance variable here should be `@books` not `@book`, as a result, an error is thrown when option 5 is selected in the user interface. Kindly fix this. :pray: 

![Screenshot from 2021-11-20 00-09-29](https://user-images.githubusercontent.com/64894927/142692148-c40d47fe-427d-4d01-acd2-0348d6b6839a.png)

Reply:
Done ✔️

Reply:
Well done! :clap: :clap: :clap: